### PR TITLE
#165: only show loading footer when loading

### PR DESCRIPTION
**Pull request by aminahbl**

### CODE REVIEW

**Comment by aminahbl** [comment]
Hi @TrebuhD I'm not sure if #165 was already resolved by https://github.com/BuddhaNexus/buddhanexus-frontend-next/pull/171. If so, this can be closed, if not maybe the conflict can be resolved.

File: features/globalSearch/GlobalSearchDesktop.tsx [hunk 1]
```diff
@@ -42,7 +42,8 @@ const GlobalSearchDesktop = () => {
     >
       <IconButton
         color="inherit"
-        aria-label={t("search.toggleSearch")}
+        // i18n leads to unknown server/client mismatch error
```
Comment (TrebuhD) [inline features/globalSearch/GlobalSearchDesktop.tsx:45]: I've been seeing this one, does this change solve it? I'm a bit stumped by this error, unfortunately!

**Review by TrebuhD — COMMENTED** [review]
I'm getting this behavior on this branch. 
https://github.com/user-attachments/assets/c54587d5-f8ce-4da0-a4ed-8bca0c5d0f4b

On the `main` branch there's no issue like this (I checked http://localhost:3000/db/pli/mn65/text). Since it seems solved for the text view,  I would suggest to close this PR and use the solution from the main branch for the Table view. What do you think @aminahbl ?

**Comment by TrebuhD** [comment]
Sorry, the video didn't attach correctly before:
https://github.com/user-attachments/assets/84b873c2-b0e2-4754-8540-eb531a92fd49

**Pull request closed without merging**